tldw_Server_API/app/main.py

```
@@ -294,6 +296,264 @@ async def _run_coordinated_shutdown(
+async def _stop_registered_job_pollers(
+    app: FastAPI,
+    handles: list[_ManagedJobPoller],
+) -> None:
+    """Stop registered job pollers, preferring explicit stop events."""
+    async def _await_job_poller_shutdown(handle: _ManagedJobPoller) -> None:
+        try:
+            await asyncio.wait_for(asyncio.shield(handle.task), timeout=handle.timeout_sec)
+        except asyncio.CancelledError:
+            pass
+        except asyncio.TimeoutError:
+            logger.warning(
```

Ruff (0.15.10)

[error] Too many arguments for `logging` format string

(PLE1205)